src/editor/index.css

```diff
@@ -282,6 +282,7 @@ li p .ag-hide:first-child {
 p:not(.ag-active)[data-role="hr"] {
   position: relative;
   color: transparent;
+  padding: 0.8em 0;
```

Comment:
@fxha little confused about why add `.8em` padding top and bottom?

Reply:
I don't know why, but exported PDF and HTML files does not have padding.